diff -ur xmlrpc-epi-0.51-old/src/xml_element.c xmlrpc-epi-0.51-new/src/xml_element.c
--- xmlrpc-epi-0.51-old/src/xml_element.c	2008-08-21 13:50:43.000000000 -0700
+++ xmlrpc-epi-0.51-new/src/xml_element.c	2008-08-21 13:50:21.000000000 -0700
@@ -170,7 +170,10 @@
 
       Q_Destroy(&root->children);
       Q_Destroy(&root->attrs);
-      my_free((char*)root->name);
+      //my_free((char*)root->name);
+	  // the above doesn't build in gcc-4 -- error: invalid lvalue in assignment
+	  // but the below does:
+      my_free(root->name);
       simplestring_free(&root->text);
       my_free(root);
    }
